sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1463,7 +1463,14 @@ def init(self):
         # Initialize xcvr table helper
         xcvr_table_helper = XcvrTableHelper()
 
-        self.load_media_settings()
+        fastboot_tbl = swsscommon.Table(xcvr_table_helper.state_db_host, 'FAST_REBOOT')
+        keys = fastboot_tbl.getKeys()
+
+        if "system" in keys:
```

Comment:
if "FAST_REBOOT" key has a expiry then we should think about following two scenarios where Xcvrd restart happens after the key has expired:

1. Xcvrd is crashing continiously
2. DRI restarts the PMON/xcvrd

Reply:
In error cases like these, wouldn't it be better to not do this optimization and take the cold-reboot route?

Reply:
yes, i agree these fall under cold boot case.